lib/updateFacebookCache.js

```diff
@@ -1,15 +1,30 @@
 var axios = require('axios');
 var config = require('../config/config.json');
 
-// update facebook's cache with most recent metadata
-function updateFacebookCache ( url ) {
-	axios.post('https://graph.facebook.com', { id: url, scrape: true, access_token: config.fbAccessToken })
-	  .then(function (response) {
-	    console.log(response);
-	  })
-	  .catch(function (error) {
-	    console.log(error);
-	  });
+function updateFacebookCache (schema) {
+
+	schema.Metadata.findAll({
+	    attributes: [
+	        [Sequelize.fn('DISTINCT', Sequelize.col('url')) ,'url'],
```

Comment:
`Sequelize` won't exist in this context -- maybe also make it an argument for `updateFacebookCache`